src/nodejs/lambda-layers/message-util/src/templates/new-submission.js

```diff
@@ -0,0 +1,35 @@
+const getNewSubmissionTemplate = (params) => {
+  const newSubmissionText = `Hello ${params.user.name},\n\nThank you for creating a request to the ${params.eventMessage.daac_name} in Earthdata Pub. On your personal dashboard (https://pub.earthdata.nasa.gov/dashboard), your submission has received the following temporary name:\n\n${params.eventMessage.submission_name} (${params.eventMessage.submission_id})\n\nPlease click on the green button on the far right of your submission's row to complete the appropriate form, which will gather information to continue your submission.\n\nThank you for using Earthdata Pub.`;
```

Comment:
Revisiting this PR now that it can be tested in SIT.

- The URLs should all be populated by an environment variable instead of being hardcoded to PROD.
- After `following temporary name:` the submission name + submission id is referenced; however, the dashboard page only references the submission name, so submission id can be removed from there.